.github/workflows/ci.yaml

```
@@ -31,6 +31,10 @@ jobs:
             ~/.cache/bazel-repo
           key: bazel-cache-${{ hashFiles('**/BUILD.bazel', '**/*.bzl', 'WORKSPACE') }}
           restore-keys: bazel-cache-
+      # TODO: make a util rule that creates docker tarball out of OCI image so we don't need skopeo
```

TODO without an owner. Either create an issue to track it, or if you want to personally track it:
```suggestion
      # TODO(thesayyn): make a util rule that creates docker tarball out of OCI image so we don't need skopeo
```

---

this is addressed via #30